openhcl/page_pool_alloc/src/lib.rs

```diff
@@ -232,7 +290,7 @@ impl SharedPoolAllocator {
 }
 
 #[cfg(feature = "vfio")]
-impl user_driver::vfio::VfioDmaBuffer for SharedPoolAllocator {
+impl user_driver::vfio::VfioDmaBuffer for PagePoolAllocator {
     fn create_dma_buffer(&self, len: usize) -> anyhow::Result<user_driver::memory::MemoryBlock> {
```

Comment:
Just for awareness, this may conflict with @eric135's change. We'll see which change goes in first...

Reply:
I expect his change to go in first, so I'll deal with the rebase.